docs/pages/linux-demo.mdx

```diff
@@ -44,38 +44,108 @@ If you want to get a feel for Teleport commands and capabilities without setting
 up any infrastructure, take a look at the browser-based [Teleport
 Labs](https://goteleport.com/labs).
 
-- A Linux host with only port `443` open to ingress traffic. You must be able
-  to install and run software on the host. Either configure access to the host
-  via SSH for the initial setup (and open an SSH port in addition port `443`)
-  or enter the commands in this guide into an Amazon EC2 [user data
+You can follow this guide with either a remote virtual machine (e.g., an Amazon
+EC2 instance) or a local Docker container. Make sure you have met the following
+requirements for your platform:
+
+<Tabs>
+<TabItem label="Remote virtual machine">
+
+- A Linux host with only port `443` open to ingress traffic. You must be able to
+  install and run software on the host. Either configure access to the host via
+  SSH for the initial setup (and open an SSH port in addition to port `443`) or
+  enter the commands in this guide into an Amazon EC2 [user data
   script](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html),
   Google Compute Engine [startup
-  script](https://cloud.google.com/compute/docs/instances/startup-scripts),
-  or similar.
+  script](https://cloud.google.com/compute/docs/instances/startup-scripts), or
+  similar.
 
   For a quick demo environment you can use to follow this guide, consider
```

Comment:
Should we remove the Digital Ocean callout here? It's not well-maintained and I think we might be giving the user too many options here.